test/unit/jquery/effects.js

```diff
@@ -10,7 +10,7 @@ QUnit.test( "jQuery.easing", function( assert ) {
 
 	assert.expect( 4 );
 
-	jQuery.easing.testOld = function( p, n, firstNum, diff ) {
+	jQuery.easing.testOld = function( _p, _n, _firstNum, _diff ) {
```

Comment:
Perhaps we should just delete the arguments since they're not used.

Reply:
I tried that, but the number of arguments matters for the test.

Reply:
I'll add a comment.